src/core/include/openvino/op/op.hpp

```diff
@@ -33,19 +33,14 @@ namespace op {
 /// \brief Root of all actual ops
 /// \ingroup ov_ops_cpp_api
 class OPENVINO_API Op : public Node {
+public:
+    OPENVINO_SUPPRESS_SUGGEST_OVERRIDE_START
```

Comment:
maybe we can insert `OPENVINO_SUPPRESS_SUGGEST_OVERRIDE_START` and `OPENVINO_SUPPRESS_SUGGEST_OVERRIDE_END` directly to `OPENVINO_RTTI_BASE`?

BTW, Why this class is base one? I supposed that `Node` should be base one?

Reply:
But then it doesn't suppress it on clang 17.0.6 I use locally. I compared preprocessor outputs for both solution inside and outside. These identical apart from line numbers - but only the later works. It's not an issue with G++ .. do you have any clue?

Reply:
Look my first commit https://github.com/openvinotoolkit/openvino/commit/f6096002894f74643752a773178af5c1a3599c40

Reply:
> BTW, Why this class is base one? I supposed that `Node` should be base one?

I didn't meant to change current state. Abstract `get_type_info` in `Node` at least forces its override elsewhere.